platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-form.html

```diff
@@ -2,9 +2,10 @@
   <md-toolbar md-scroll-shrink>
     <div class="md-toolbar-tools">
       <h3>
-        <span ng-if="$ctrl.isNew">New Token</span>
+        <span ng-if="$ctrl.isNew && $ctrl.user.is_active">New Token</span>
```

Comment:
This just controls the display of this title; maybe we should assume that you can't even make it to this form for a deactivated user? (i.e. add a check when the form loads up, and redirect to home). Unless I'm missing something and an admin should need to see this form for deactivated users? (in which case maybe we need an alternate title, or maybe the badge below is enough).